source/Fleet.cpp

```diff
@@ -333,7 +355,7 @@ Fleet::Variant::Variant(const DataNode &node)
 	weight = 1;
 	if(node.Token(0) == "variant" && node.Size() >= 2)
 		weight = node.Value(1);
-	else if(node.Token(0) == "add" && node.Size() >= 3)
+	else if((node.Token(0) == "add" || node.Token(0) == "remove") && node.Size() >= 3)
```

Comment:
I had added it to account for that possibility. The function i wrote ultimately decrements the total by the weight of the variant being removed, and not the weight specified in the `remove variant` node (if any).

I can remove this check if desired.